src/main/thrift/parquet.thrift

```diff
@@ -380,6 +410,38 @@ struct JsonType {
 struct BsonType {
 }
 
+/** Physical type and encoding for the geometry type */
+enum GeometryEncoding {
+  /**
+   * Allowed for physical type: BYTE_ARRAY.
+   *
+   * Well-known binary (WKB) representations of geometries.
+   */
+  WKB = 0;
+}
+
+/** Interpretation for edges of elements of a GEOMETRY type */
+enum Edges {
+  PLANAR = 0;
+  SPHERICAL = 1;
+}
+
+/**
+ * GEOMETRY logical type annotation (added in 2.11.0)
+ *
+ * GeometryEncoding and Edges are required. CRS is optional.
+ *
+ * Once CRS is set, it MUST be a key to an entry in the `key_value_metadata`
+ * field of `FileMetaData`.
+ *
+ * See LogicalTypes.md for detail.
+ */
+struct GeometryType {
+  1: required GeometryEncoding encoding;
```

Comment:
What is the expectation for reading this? What happens if there is an unknown value in `GeometryEncoding`? Will readers be required to fail?

Reply:
I think it is a reasonable expectation that: the reader should fail on unknown value.

Reply:
Make sense. I have added a statement for the expectation of both writer and reader implementations for the required `GeometryEncoding` and `Edges` fields.